Correct OutlineBorder gapStart when InputDecoration icon is specified

HansMuller opened the pull request:
Fixes https://github.com/flutter/flutter/issues/18111

Golden images commit: https://github.com/flutter/goldens/pull/1/commits/299210d0a2eb639bebdcbfaf9a8cc668bdf905f3

HansMuller committed:
Correct OutlineBorder gapStart when InputDecoration icon is specified

HansMuller committed:
Fixed a lintO

HansMuller committed:
Fixed a lintO

tvolkert reviewed (commented):
Tests are failing on Linux - do they pass locally for you?  If not, you can update the goldens again with `--update-goldens`

HansMuller committed:
Updated goldens.version

HansMuller commented:
I think I've sorted out the problem.

- The test does indeed fail locally although the image diff is inexplicable since the test only depends on widgets/cupertino and my change was to the material library.
- It appears that @xster updated the goldens repo last week, but did not land the corresponding change: https://github.com/flutter/goldens/commit/a30f6886b4498484f1dfd799fd97a8ea6f162c18
- So when I added new golden images for this PR, I ended up with the new cupertino navbar goldens as well. 

I've updated the nav_bar goldens and all appears to be well.

HansMuller committed:
fix version commit hash

HansMuller committed:
skip tests, not expectations
